docs/pbc/all/search/202410.0/base-shop/third-party-integrations/algolia/algolia-personalization.md

```diff
@@ -0,0 +1,355 @@
+        ];
+}
+```
+
+4. Build JavaScript assets for Yves using one of the following commands:
+```bash
+npm ci && npm run yves
+```
+
+```bash
+console frontend:project:install-dependencies && console frontend:yves:build
+```
+
+5. Install required `search-insights` dependency:
+```bash
+npm i search-insights
+```
+
+### Check your Yves's compatibility with new functionality
+
+In case of customizations, your codebase could have updated Yves templates on the project level (`src/Pyz/Yves/`).
+It could be a reason that some events won't be triggered or triggered with incorrect data.
+
+#### Run the project
+1. Set `TraceableEventWidgetConfig::isDebugEnabled()` set to `true`.  
+2. Run the project locally or deploy to testing environment.
+3. Open your Shop Storefront home page.
+4. Open browser's development console and check "Preserve log" in the settings.
+
+#### Check triggered events and their payload
+
+When debug mode is enabled, you can see event logs in the console to help with inspection.
+
+Monitor the browser's console and execute following cases:
+* Open the home page.
+  * If the home page has products, click on a product - `PRODUCT_CLICK`.
+  * If the home page has the add to cart button, click on it - `ADD_TO_CART`
+* Open a product's details page, you should see events for the actions:
+  * `PAGE_LOAD` with the product's SKU.
+  * `ADD_TO_CART` with the product's SKU, currency, price and quantity, when user clicks **Add to cart**.
+  * `ADD_TO_SHOPPING_LIST` with product SKU when user clicks **Add to shopping list**.
+  * `ADD_TO_WISHLIST` with product SKU when user clicks **Add to wishlist**.
+* Open a **Category** or **Search Results** page:
+  * `QueryID` should be present in the event payload on this page type.
+  * `PAGE_LOAD` with displayed products SKUs and displayed search filters.
+  * `PRODUCT_CLICK` when user clicks on results.
+  * `ADD_TO_CART` with product SKU, currency and price, when user clicks Add to cart from the catalog page.
+  * `FILTER_CLICK` with filters list, when user clicks any filter from the filter section.
+* Open Cart page
+  * (if applicable) add a new product from add to cart widget `ADD_TO_CART`
+  * (if applicable) save cart items to a shopping list `ADD_TO_SHOPPING_LIST`
+* Open Quick Order page
+  * (if applicable) add a new product from add to cart widget `ADD_TO_CART`
+  * (if applicable) save cart items to a shopping list `ADD_TO_SHOPPING_LIST`
+* Open Order Success page
+  * `PAGE_LOAD` with currency, order total, skus, prices, quantities  of purchased products.
+
+To view a full list of available events, refer to the `traceable-events-algolia` [Readme file](https://github.com/spryker-shop/traceable-event-widget/src/SprykerShop/Yves/TraceableEventWidget/Theme/default/components/molecules/traceable-events-algolia/README.md).
+
+
+#### Common Issues and Solutions
+
+##### Prerequisites
+
+If you need to add, modify, or fix events at the project level, start with these two steps:
```

Comment:
Would it be correct to say that most solutions in this section are related to adding, changing, or fixing events?